flake.nix

```diff
@@ -39,6 +39,24 @@
               clang = callPackage ./mods/Windows/OnlineCTR/Network_PC/Server { ctrModSDK = self; stdenv = clangStdenv; trustCompiler = true; inherit withDebug; };
             };
           };
+          mkOnlineClient = withDebug: {
+            native32 = with pkgs32; {
+              gcc = callPackage ./mods/Windows/OnlineCTR/Network_PC/Client { ctrModSDK = self; inherit withDebug; };
+              clang = callPackage ./mods/Windows/OnlineCTR/Network_PC/Client { ctrModSDK = self; stdenv = clangStdenv; inherit withDebug; };
+            };
+            arm32 = with pkgsARM32; {
+              gcc = callPackage ./mods/Windows/OnlineCTR/Network_PC/Client { ctrModSDK = self; inherit withDebug; };
+              clang = callPackage ./mods/Windows/OnlineCTR/Network_PC/Client { ctrModSDK = self; stdenv = clangStdenv; inherit withDebug; };
+            };
```

Comment:
You can drop these lines (43-50).

Reply:
I thought to leave it for when the client is ported to linux, but I'm fine either way.